presto-product-tests/bin/locations.sh

```diff
@@ -2,3 +2,16 @@ SCRIPT_DIR=${BASH_SOURCE%/*}
 PRODUCT_TESTS_ROOT="${SCRIPT_DIR}/.."
 PROJECT_ROOT="${PRODUCT_TESTS_ROOT}/.."
 DOCKER_CONF_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker"
+
+# Yes, JDBC_DRIVER_JAR is intentionally set to a directory. 
+# This works better than leaving it empty or unset.
+# The open source Presto JDBC driver is included in the fat jar with the tests.
+JDBC_DRIVER_JAR="${PRODUCT_TESTS_ROOT}"
+TEMPTO_CONFIG_YAML="${PRODUCT_TESTS_ROOT}/conf/tempto/tempto-configuration-for-docker-default.yaml"
+TEMPTO_CONFIG_KERBEROS_YAML="${PRODUCT_TESTS_ROOT}/conf/tempto/tempto-configuration-for-docker-kerberos.yaml"
+
+# If you are using the Teradata/Simba JDBC driver, uncomment the next three lines.
+#
+#JDBC_DRIVER_JAR="${PROJECT_ROOT}/simba/PrestoJDBC_1.0.9.1017/PrestoJDBC42.jar"
+#TEMPTO_CONFIG_YAML="${PRODUCT_TESTS_ROOT}/conf/tempto/tempto-configuration-for-docker-simba.yaml"
+#TEMPTO_CONFIG_KERBEROS_YAML="${PRODUCT_TESTS_ROOT}/tempto-configuration-for-docker-kerberos-simba.yaml"
```

Comment:
Please remove the commented-out code, along with the comment. Please document the PRESTO_JDBC_DRIVER_JAR env variable in README.md, near the info on PRODUCT_TESTS_JAR et al.